src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor

```diff
@@ -112,7 +112,9 @@
             <div class="d-flex align-end" style="width:100%;">
                 @if (_appFormModel.HasValue)
                 {
-                    <MButton Height="40" OnClick="RemoveAppAsync" Width=100 Class="rounded-pill" Text Color="red" Style="float:left;">@T("Delete")</MButton>
+                    <MButton OnClick="RemoveAppAsync" Left Icon Color="#FF5252" Depressed Class="delete-btn">
```

Comment:
in stack componment add SDeleteButton

Reply:
already submit pr to MASA.Stack.Component